### DISCUSSION TITLE
Assigment 2

### DISCUSSION THREAD

**Discussion by Memla123** [irgroup-classrooms/dis08-2021#49] [Q&A]
Hallo 
leider komme ich  nicht in meine Gruppe rein, da meine gruppen Mitglieder doppelt in der Gruppe drin sind und wir leider nicht wissen wie man die doppelten weg bekommt.

**Comment by phschaer** [comment]
Wenn Sie mir sagen, um welche Gruppe es geht und welches Mitglied in der Gruppe doppelt vertreten ist, kann ich mir das gerne ansehen.

**Reply by Memla123** [reply]
bei der Gruppe handelt es sich um Data deputy und die Mitgleider die doppelt sind heißen DieuLinhThuy und simsek-beyza.

**Reply by phschaer** [reply]
Ich habe Sie händisch zum Team hinzugefügt. Falls noch jemand fehlt, bitte nochmal ein Ticket aufmachen.
